source/cloud-native-platform/path-to-live/shutter.html.md.erb

```diff
@@ -1,54 +1,126 @@
+
+### Shuttering A record types
+
+A record types have some specific requirements with regard to shuttering. A records typically only allow IP addresses to be used BUT within Azure it is possible to direct A records to Azure resources, known as alias records.
+This means we can direct A record types to Azure services such as Azure Front Door or Azure Web Apps.
+
+The way DNS has been setup for this project, an A record can have one of the following set:
 
-`shutter: true` will enable the shutter page and redirect all the traffic to Azure CDN.
-`shutter: false` will disable the shutter page and redirect all the traffic back to Azure Frontdoor.
+- **record** - either a single IP or a list of IP addresses
+- **shutter_resource_id** - an Alias to an Azure resource specifically used for shuttering
+- **alias_target_resource_id** - an Alias to an Azure resource if not IP is used, can also be used for shutting but preference should be shutter_resource_id
 
-This is a self service request and does not require Platform Operations to create, approve or merge.
-Therefore this can be done out of hours without having Platform Operations around for shuttering only.
+For shuttering purposes if you need to shutter an A record it must> have one of the following settings in the A record configuration:
+
+- **shutter_resource_id**
+- **alias_target_resource_id**
+
+This is a requirement and failing to set one of these values on your A record before shuttering will result in the CI pipelines failing and your site may be in a limbo state until reverted or updated and re-run.
+
+### Shuttering CNAME record types
+
+CNAME records are much easier to shutter, for each record an equivalent shutter config must exist, as detailed in the Enable shuttering with a DNS swap section.
+
+Simply enable or disable the shutter using True/False values
 
 ## Shuttering Errors
 
 Shuttering error message - 404 Not Found - we couldn't find the page, please check the URL and try again - this occurs when browsing to folders after the main URL, which have not been set up for shuttering. Therefore teams need to update their custom shutter page.  
 
-See [azure-shutter-pages](https://github.com/hmcts/azure-shutter-pages) repository for examples, where there is shutter pages for further directories beyond the main URL.  For example: [et-pet](https://github.com/hmcts/azure-shutter-pages/tree/master/et-pet)
\ No newline at end of file
+See [azure-shutter-pages](https://github.com/hmcts/azure-shutter-pages) repository for examples, where there is shutter pages for further directories beyond the main URL.  For example: [et-pet](https://github.com/hmcts/azure-shutter-pages/tree/master/et-pet)
+
+
+
+
+
+Use the following steps to enable shuttering:
+
+Clone the azure-public-dns repository
```

Comment:
do we want to number the steps?

Reply:
Good spot, those shouldnt have been in that file, they were copied from my notes.
Remove and numbering added to main steps.